src/Contao/ContaoTranslatorService.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+/**
+ * This file is part of contao-community-alliance/translator.
+ *
+ * (c) 2013-2016 Contao Community Alliance <https://c-c-a.org>
+ *
+ * This project is provided in good faith and hope to be usable by anyone.
+ *
+ * @package    contao-community-alliance/translator
+ * @author     Sven Baumann <baumann.sv@gmail.com>
+ * @copyright  2013-2016 Contao Community Alliance <https://c-c-a.org>
+ * @license    https://github.com/contao-community-alliance/translator/LICENSE LGPL-3.0+
+ * @link       https://github.com/contao-community-alliance/translator
+ * @filesource
+ */
+
+namespace ContaoCommunityAlliance\Translator\Contao;
+
+use ContaoCommunityAlliance\Translator\TranslatorChain;
+use ContaoCommunityAlliance\Translator\TranslatorInitializer;
+use Symfony\Component\EventDispatcher\EventDispatcherInterface;
+
+/**
+ * This class create the contao translator service.
+ */
+class ContaoTranslatorService
```

Comment:
This class is actually a factory class. So I prefer naming it ContaoTranslatorFactory instead.